packages/lib/server-only/pdf/insert-field-in-pdf.ts

```diff
@@ -12,7 +12,10 @@ import { isSignatureFieldType } from '@documenso/prisma/guards/is-signature-fiel
 import type { FieldWithSignature } from '@documenso/prisma/types/field-with-signature';
 
 export const insertFieldInPDF = async (pdf: PDFDocument, field: FieldWithSignature) => {
-  const fontCaveat = await fetch(process.env.FONT_CAVEAT_URI).then(async (res) =>
+  const fontDancingScriptURL =
```

Comment:
Ideally we do the same trick we did for Caveat since going to an external source is costly

Reply:
okay sure, I'll pull dancing script locally and we can use it